lib/record_store/zone/config.rb

```diff
@@ -8,7 +8,10 @@ class Config
       validate :validate_zone_config
 
       def initialize(ignore_patterns: [], providers: nil, supports_alias: nil)
-        @ignore_patterns = ignore_patterns
+        @ignore_patterns = ignore_patterns.map do|ignore_pattern|
```

Comment:
There's a space missing between `do` and `|`. Sorry :-/

Reply:
I've seen this pattern in may different places in your code. Maybe there's something in your editor. Maybe that's good 🤷‍♂️  and I'm just not used to that.

Reply:
Will update it if it devistes from the rest of the code.  If there's a linter config, I'll update it to enforce this.